filament/src/RenderPass.cpp

```diff
@@ -639,11 +756,23 @@ void RenderPass::Executor::recordDriverCommands(FEngine& engine, backend::Driver
 
             pipeline.program = ma->getProgram(info.materialVariant);
 
-            if (uboDataIndex != info.index) {
+//            if (uboDataIndex != info.index)
```

Comment:
Remove?

Reply:
yes, in fact this wasn't a good optimization because the backends should already do this (at least the GLES one does), and the backend knows better if this operation is heavy or not (it shouldn't).